server/db/models/OrderProduct.js

```diff
@@ -0,0 +1,24 @@
+const db = require('../index');
+const Sequelize = require('sequelize');
+
+const OrderProduct = db.define('orderproduct', {
+    id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true,
+    },
+    productId: {

```

Comment:
Lets sit down and look at this - something is going wrong.

The reason this is a big deal - is if for some reason sequelize is failing to set these relations on its own - then these relations are not being enforced (the proper SQL term is actually `constrained`) which is the entire benefit.